src/styles/variables.scss

```diff
@@ -136,3 +136,6 @@ $annotator-border-radius: 4px;
 
 $annotator-shadow--sidebar: 0px 1px 4px color.scale($color-shadow, $alpha: -50%);
 $annotator-timing--sidebar-collapse-transition: 150ms;
+
+//test for highlight colors per user
+$user-colors: #ff0a0a, #20e9ff, #447fff, #fff400, #8335ff, #2fffa2;
```

Comment:
Colors look nice. Some of them may have issues for low vision people (I'm mostly concerned with being able to see a difference between regular and highlighted text, as we should have another indicator of who left an annotation that they can access, and choosing many colors that are very distinct is pretty much impossible), but we can easily change the list in the future.

Reply:
We could also randomly generate the highlight colors, probably using hsla(). This is something to consider next quarter.